lms/templates/components/card/card.underscore

```diff
@@ -2,40 +2,40 @@
 <div class="wrapper-card-core">
     <div class="card-core">
         <% if (pennant) { %>
-            <small class="card-type"><%- pennant %></small>
+            <small class="card-type"><%= HtmlUtils.escape(pennant) %></small>
         <% } %>
         <h3 class="card-title"
             <% if (!_.isUndefined(srInfo)) { %>
                 aria-describedby="<%= srInfo.id %>"
             <% } %>
-            ><%- title %>
+            ><%= HtmlUtils.escape(title) %>
```

Comment:
I agree that HtmlUtils.escape(myHtml) is very confusing-- I didn't understand it myself. Like @andy-armstrong said, it seems like it is wrong because why would I wan to to escape something that is supposed to be HTML?

If it's expected to be HTML, why do we have to call any method (obviously STILL confused)?